gammapy/astro/darkmatter/tests/test_utils.py

```diff
@@ -33,5 +37,6 @@ def test_dmfluxmap(jfact):
         jfact * diff_flux.integral(energy_min=energy_min, energy_max=energy_max)
     ).to("cm-2 s-1")
     actual = int_flux[5, 5]
-    desired = 1.9483e-12 / u.cm**2 / u.s
+    print(actual)
```

Comment:
Please remove print statement.

Reply:
Thanks, @registerrier. I removed the print statements